central/networkgraph/entity/datastore/datastore_impl.go

```diff
@@ -231,6 +232,10 @@ func (ds *dataStoreImpl) GetAllMatchingEntities(ctx context.Context, pred func(e
 	return entities, nil
 }
 
+func (ds *dataStoreImpl) GetEntityByQuery(ctx context.Context, query *v1.Query) ([]*storage.NetworkEntity, error) {
+	return ds.storage.GetByQuery(ctx, query)
```

Comment:
There should probably be some error handling here, even though there is error handling where it is called.

Reply:
Why?  Basically just a pass through function.